src/components/content-tab-sync.tsx

```diff
@@ -162,23 +163,33 @@ function NoAuthSyncTab() {
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function ContentTabSync( { selectedSite }: { selectedSite: SiteDetails } ) {
 	const { __ } = useI18n();
-	const { syncSites, connectedSites, setConnectedSites, isFetching } = useSyncSites();
+	const { connectedSites, connectSite, disconnectSite } = useSiteSyncManagement();
+	const { syncSites, isFetching } = useFetchWpComSites( connectedSites );
```

Comment:
We are already calling `useFetchWpComSites` inside `useSiteSyncManagement`. Could we remove the fetch from this component in favor of the call inside the hook?

https://github.com/Automattic/studio/blob/de20256c1fdfcc066d3988e25ac18a23c0073011/src/hooks/use-site-sync-management.ts#L10

Something like: 

```suggestion
	const { connectedSites, connectSite, disconnectSite, syncSites, isFetching } = useSiteSyncManagement();
```

Reply:
Updated in https://github.com/Automattic/studio/pull/610/commits/91efc5acbb687e0e33a892b75764508e13549f13